sparkles/core.py

```diff
@@ -1142,11 +1151,27 @@ def check_guide_count(self):
                 f'{obs_type} count of 9th ({mag9:.1f} for {self.guides.t_ccd:.1f}C) '
                 f'mag guide stars {self.guide_count_9th:.2f} < {count_9th_lim}')
 
-        count_lim = 4.0 if self.is_OR else 6.0
-        if np.round(self.guide_count, decimals=2) < count_lim:
+        # Rounded guide count
+        guide_count_round = np.round(self.guide_count, decimals=2)
+
+        # Set critical guide_count threshold
```

Comment:
Make this comment explain *why* this code is here, not *what* it does (which is already clear). Include a reference to the SSAWG page which discusses and approves this change.